packages/fxa-content-server/app/scripts/templates/partial/account-suggestion.mustache

```diff
@@ -1,6 +1,10 @@
 {{#showAccountSuggestion}}
```

Comment:
The SVG is different because I'd changed the `fill` setting from a specific RGB to `currentColor`. I'll discard this change since it didn't end up working to manipulate the fill color with TW.